target/linux/ath79/nand/base-files/etc/board.d/02_network

```diff
@@ -11,6 +11,10 @@ ath79_setup_interfaces()
 	aerohive,hiveap-121)
 		ucidef_set_interface_lan "eth0"
 		;;
+	domywifi,dw33d)
+		ucidef_add_switch "switch0" \
+			"0@eth1" "2:lan" "3:lan" "4:lan" "5:lan" "6@eth0" "1:wan"
```

Comment:
Is this affected by eth0/eth1 swap or why are you deviating from ar71xx here?

Reply:
Yes, the eth0/ethe port changes.